openpgp.d.ts

```diff
@@ -337,10 +337,9 @@ export class Message<T extends MaybeStream<Data>> {
 }
 
 /* ############## PACKET #################### */
-
-export declare abstract class BasePacket {
+export declare abstract class BasePacket<AsyncRead extends boolean = false> {
   static readonly tag: enums.packet;
-  public read(bytes: Uint8Array): void;
+  public read(bytes: Uint8Array): AsyncRead extends true ? Promise<void> : void;
```

Comment:
Maybe we can avoid this complexity and just mark all of the `read` functions `async`?

Reply:
Breaking change, and also confusing